src/endpoints/test.ts

```diff
@@ -0,0 +1,43 @@
+import { config } from 'dotenv';
+import { join } from 'path';
+import Client from '../index';
+
+config({ path: join(__dirname, '.env.e2e') });
+
+// Extract and validate required environment variables
+const token = 'jKD7wagy7xxHAw.atlasv1.OgOqpRWWNPU5hlzeNvbHXLWnFkxMMFceW9m5LYoyUDzyYoSytwfphRk4V5yu6CYNFCA'
+const organizationName = 'jwalab'
+const client = new Client({
+  token,
+});
+
+describe('listTags', () => {
```

Comment:
there is a test file already, we should add the tests to it and later we can split for each endpoints if it becomes too hard to maintain